data/settings.js

```diff
@@ -6583,7 +6583,20 @@ imapc_sasl_mechanisms {
   plain = yes
   login = yes
 }
-\`\`\``
+\`\`\`
+
+Supported mechanisms are:
+
+ * ANONYMOUS
+ * EXTERNAL
+ * LOGIN
+ * OAUTHBEARER
+ * PLAIN
+ * SCRAM-SHA-1
+ * SCRAM-SHA-1-PLUS
+ * SCRAM-SHA-256
+ * SCRAM-SHA-256-PLUS
+ * XOAUTH2`
```

Comment:
Should this be a table where the other column is explanation of imapc_password field? Like it being ignored with anonymous and external. Also I'm not sure what it is with oauth - the token?

Reply:
for oauth password is access token